include/fault.h

```diff
@@ -0,0 +1,91 @@
+#ifndef FAULT_H
+#define FAULT_H
+
+#include "padmgr.h"
+
+// These are the same as the ansi color codes
+#define FAULT_COLOR_BLACK      0
+#define FAULT_COLOR_RED        1
+#define FAULT_COLOR_GREEN      2
+#define FAULT_COLOR_YELLOW     3
+#define FAULT_COLOR_BLUE       4
+#define FAULT_COLOR_MAGENTA    5
+#define FAULT_COLOR_CYAN       6
+#define FAULT_COLOR_WHITE      7
+#define FAULT_COLOR_DARK_GRAY  8
+#define FAULT_COLOR_LIGHT_GRAY 9
```

Comment:
This is correct, it indexes into `FaultDrawer.printColors` which the last two entries are:
https://github.com/Thar0/oot/blob/2c29cbe3576dc3380f435629926890c9a305cf06/src/code/fault_drawer.c#L94-L95
```c
        GPACK_RGBA5551(120, 120, 120, 1),
        GPACK_RGBA5551(176, 176, 176, 1),
```

While on topic, does anyone have a source for
```c
#define VT_COLOR_LIGHTGRAY     8
#define VT_COLOR_DARKGRAY      9
```
in vt.h? It is not mentioned on https://en.wikipedia.org/wiki/ANSI_escape_code#3-bit_and_4-bit
Also at least Windows terminal doesn't seem to support those

Reply:
our vt macros were made years ago by Random i believe, not sure hed remember

Reply:
To be clear, I mentioned this was reversed since the comment above states they are the same as the ansi color codes which is what we have `vt.h` for, so they should probably be consistent if that statement is right

Reply:
I've changed PURPLE to MAGENTA in vt.h since magenta is the correct name. I've clarified that dark gray and light gray are specific to fault and are not ANSI. I've removed dark gray and light gray from vt.h since these do not exist, and using them would be erroneous (the game never uses them)